src/runtime/fetch/call.ts

```diff
@@ -14,6 +14,8 @@ export interface CallContext {
   body?: any
 }
 
+const BINARY_CONTENT_TYPE_RE = /^(audio|font|video|image\/(?!svg)|application\/(?!json))/
```

Comment:
Text mimes are more limited. We could inverse to list `text/*` + `application/json` as text content and assume the rest as binary. (maybe + empty content-type as text?)